code/modules/status_system/statusEffects.dm

```diff
@@ -1308,37 +1311,80 @@
 			gang = null
 
 		onUpdate(timePassed)
-			var/area/cur_area = get_area(H)
-			if (cur_area?.gang_owners == gang && prob(50))
-				on_turf = 1
-
-				//get distance divided by max distance and invert it. Result will be between 0 and 1
-				var/buff_mult = round(1-(min(GET_DIST(owner,gang.locker), max_dist) / max_dist), 0.1)
-				if (buff_mult <=0)
-					buff_mult = 0.1
+			var/mob/living/carbon/human/H
+			if(!ishuman(owner))
+				return
+			H = owner
+			elapsedTime += timePassed
+			var/times = (elapsedTime / tickSpacing)
+			if(times >= 1)
+				elapsedTime -= (round(times) * tickSpacing)
+				for(var/i in 1 to times)
+					H.HealDamage("All", 0.2, 0.2, 0)
```

Comment:
Not a huge fan of magical passive healing, maybe only when on low health/in crit?

Reply:
idk. admittedly it existed in the old version and i facelifted it. for now it further encourages gang members stickin near their locker?